zproject/urls.py

```diff
@@ -222,6 +222,10 @@
         {'GET': 'zerver.views.streams.list_subscriptions_backend',
          'POST': 'zerver.views.streams.add_subscriptions_backend',
          'PATCH': 'zerver.views.streams.update_subscriptions_backend'}),
+    # PATCH update stream name
+    url(r'^rename_stream$', 'zerver.lib.rest.rest_dispatch',
+        {'PATCH': 'zerver.views.streams.json_rename_stream'}),
+
```

Comment:
Yeah that would be remaining part of the PR for shifting this API(backend part).